www_src/pages/project/Positionable.jsx

```diff
@@ -0,0 +1,135 @@
+"use strict";
+
+var React = require("react");
+var classes = require("classnames");
+
+var Positionable = React.createClass({
+
+  getInitialState: function() {
+    return {
+      x: this.props.x || 0,
+      y: this.props.y || 0,
+      scale: this.props.scale || 1,
+      angle: this.props.angle || 0,
+      zIndex: (typeof this.props.zIndex !== "undefined") ? this.props.zIndex : 1,
+      interactive: (typeof this.props.interactive !== "undefined") ? this.props.interactive : true,
+      touchactive: false
+    };
+  },
+
+  componenWillMount: function() {
```

Comment:
technically, yes, but also technically, I was in the process of deleting this function and... forgot? O_o